docs/content/getting-started/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "Traefik Getting Started Quickly With Kubernetes"
+description: "Looking to get started with Traefik Proxy quickly? Read the technical documentation to learn a simple use case that leverages Kubernetes."
+---
+
+# Quick Start
+
+A Simple Use Case Using Kubernetes
+{: .subtitle }
+
+Using Traefik in a Kubernetes environment could be difficult to start with.
```

Comment:
This was probably not your intention but you agree this literally says that Traefik is difficult to use on Kubernetes?
I suggest rephrasing it so we don't scare newcomers right in the beginning 😂